.github/workflows/test.yml

```
@@ -0,0 +1,51 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        node: [10.x, 12.x, 14.x]
+        python: [2.7, 3.6, 3.7, 3.8]
+        os: [macos-latest, ubuntu-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+      - name: Use Python ${{ matrix.python }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python }}
+        env:
+          PYTHON_VERSION: ${{ matrix.python }}
+      - name: Install Dependencies
+        run: |
+          python -m pip install --upgrade pip flake8 pytest==4.6.6
+          npm install
+      - name: Lint Python
+        run: |
+          python -m flake8 --version
+          python -m flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+          python -m flake8 . --count --exit-zero --ignore=E111,E114,W503 --max-complexity=10 --max-line-length=127 --statistics
+      - name: Run Tests
+        shell: bash
+        run: |
+          echo "Testing with Python $(python --version 2>&1)"
+          npm test
+          if [ "$RUNNER_OS" == "Windows" ]; then
+            GYP_MSVS_VERSION=2015 GYP_MSVS_OVERRIDE_PATH="C:\\Dummy" python -m pytest
+          else
+            python -m pytest
+          fi
```

```suggestion
          python -m pytest
```
`actions/setup-python` will ensure that `python` points to `matrix.python-version` on all platforms.

---

hmm - 
> `E         ValueError: Could not locate Visual Studio installation.`

this appears not to be the case.